rpg/load_game_map.py

```diff
@@ -145,25 +146,27 @@ def load_maps():
     """
 
     # Directory to pull maps from
-    mypath = "resources/maps"
+    map_dir = "resources/maps"
 
-    if load_maps.map_file_names is None:
-
-        # Dictionary to hold all our maps
-        load_maps.map_list = {}
+    # Dictionary to hold all our maps
+    load_maps.map_list = {}
 
+    if LOAD_ALL_MAPS:
         # Pull names of all json files in that path
         load_maps.map_file_names = [
             f[:-5]
-            for f in os.listdir(mypath)
-            if isfile(join(mypath, f)) and f.endswith(".json")
+            for f in os.listdir(map_dir)
+            if isfile(join(map_dir, f)) and f.endswith(".json")
         ]
```

Comment:
You can leverage pathlib to make something like this more succinct:

```py
load_maps.map_file_names = [pth.stem for pth in Path(map_dir).glob("*.json")]
```

Reply:
Thanks for the suggestion! I ended up just removing this since it's unused code